BuildScripts/Common.ps1

```diff
@@ -0,0 +1,23 @@
+# if ($___COMMON__H___) { return }
+# $___COMMON__H___ = $true
+
+$RepoRoot = Split-Path $PSScriptRoot -Parent
+$ProjectPath = Join-Path $RepoRoot DXMainClient DXMainClient.csproj
+$CompiledRoot = Join-Path $RepoRoot Compiled
+$EngineMap = @{
+  'UniversalGL' = 'OpenGL'
+  'WindowsDX'   = 'Windows'
+  'WindowsGL'   = 'OpenGL'
+  'WindowsXNA'  = 'XNA'
+}
+
+function Build-Project($Configuration, $Game, $Engine, $Framework) {
+  $Output = Join-Path $CompiledRoot $Game $Framework Resources Binaries ($EngineMap[$Engine])
```

Comment:
Can you add `any` for UniversalGL?

`$path\..\Compiled\YR\net6.0\any\Resources\Binaries\OpenGL`